dependency-check-suppressions.xml

```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
 
+    <suppress>
+        <notes>
+            Commons BeanUtils: has been updated to
```

Comment:
Could you please change content of the "notes" here to mention the false positive situation? Besides, the link here will show the latest content everytime it is opened, so it may be confusing when the latest content of the page dosn't match the upgrade in the build.gradle.

Reply:
Please better examples of suppression blocks in the file, to make the wording here more clear.

Reply:
Can you please suggest  the comments ?  It will help to provide the expected comments 👍

Reply:
I struggle to figure it out .

Reply:
DONE